config/environments/development.rb

```diff
@@ -2,6 +2,9 @@
   # Settings specified here will take precedence over those in
   # config/application.rb.
 
+  # Configure the domains permitted to access coordinates API
+  config.allowed_cors_origin = "https://#{ENV.fetch('DOMAIN')}"
```

Comment:
`DOMAIN` is set up in  `config/initializers/default_url_options.rb` please use that one